src/core/HatAllocator.ts

```diff
@@ -66,25 +80,25 @@ export class HatAllocator {
     }
   }
 
-  addDecorationsDebounced() {
+  // Use default debounced delay. 60Hz
+  addDecorationsDebouncedDefault() {
+    this.addDecorationsDebounced(16);
```

Comment:
Move to constant at top?

Reply:
I mean it's only every used at one place. What would the constant improve?

Reply:
I'm not a fan of magic numbers in code, but I guess I don't feel strongly here as you have a comment right above it explaining the magic number

Reply:
I agree that unnamed and undocument that numbers is not a good thing. In this case I think we're fine due to the comment.